src/defs_version.h

```diff
@@ -49,5 +49,5 @@
 //
 #ifndef VERSION
     // In that case, this value is used
-    #define VERSION "2.1-alpha-pre"
```

Comment:
could you add a warning about this problem in the above text? Hopefully we can avoid making this mistake again :)